packages/app/scripts/start.js

```diff
@@ -170,6 +171,14 @@ function openBrowser(port, protocol) {
 }
 
 function addMiddleware(devServer, index) {
+  devServer.use(
```

Comment:
@CompuIves @lbogdan 
I don't know how many servers you are using.
From a first glimpse I think you have this webpackDevServer, an iframe with the running app, some proxies and it seems it's hosted in the cloud.

Any other CSP settings you deem necessary or any other servers?

Reply:
Am I right, that nginx is also used at prod servers?